### PR TITLE
GSoC proposal: CERNBox Notifications

### PR DESCRIPTION
**Pull request by javfg**
# 2023 GSoC Proposal for CERNBox

Adds: 
* New 2023 project: CERNBox
* New 2023 GSoC proposal: Extend and improve the CERNBox Notifications platform

### CODE REVIEW

File: _gsocproposals/2023/proposal_CERNBox-notifications.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+## Task ideas
+
+* Extend the simple notifications in the Web front-end into a notifications center
+* Implement a Mattermost notification extension
+* Provide a richer set of actions for the notification API (filters, delayed actions, metrics)
+* Create documentation for the new features
+
+Any ideas the student comes up while working on the project are welcome!
```
Comment (vvolkl) [inline _gsocproposals/2023/proposal_CERNBox-notifications.md:30]:
```suggestion
Any ideas the student comes up with while working on the project are welcome!
```

**Comment by hegner** [comment]
@javfg - would you have a second/backup supervisor?

**Comment by javfg** [comment]
> @javfg - would you have a second/backup supervisor?

Yes! I was just adding it before I read your comment. :+1:

**Comment by hegner** [comment]
> > @javfg - would you have a second/backup supervisor?
> 
> Yes! I was just adding it before I read your comment. 👍

Could you update the mentors file as well then? Thanks!

**Pull request merged**